bindings/java/iota-wallet-java/lib/native/src/lib.rs

```diff
@@ -29,11 +29,33 @@ pub extern "system" fn Java_org_iota_api_NativeApi_createMessageHandler(
     _class: JClass,
     config: JString,
 ) {
-    let manager_options: ManagerOptions = {
-        let input: String = env.get_string(config).expect("Couldn't get java string!").into();
-        serde_json::from_str(&input).unwrap()
+    let manager_options: ManagerOptions = match env.get_string(config) {
+        Ok(json_input) => match serde_json::from_str(&String::from(json_input)) {
+            Ok(manager_options) => manager_options,
+            Err(err) => {
+                env.throw_new("java/lang/Exception", err.to_string()).unwrap();
```

Comment:
Are all these `throw_new` safe to unwrap?

Reply:
Yes I would argue it the same way as for the above comment.
Maybe we should create an issue for it and let Brord have also a look, so we have 2 opinions

Reply:
Yeah sure, do that please

Reply:
oki